subtests/docker_cli/run_cidfile/run_cidfile.py

```diff
@@ -137,6 +137,8 @@ def run_once(self):
                                                lambda x: mustfail(x, 125)))
         self._check_failure_cidfile_present(containers[-1])
         # cidfile already exists (exited container)
+        # FIXME: this occasionally throws SIGPIPE, presumably because
+        #  container isn't fully ready. This is a tough one to solve.
```

Comment:
SIGPIPE in the sense that the test is sending "exit\n" to a container that isn't ready yet. I think. It's infrequent, and I don't have time right now to characterize it. I'm adding the FIXME because once we get CI working this test will start failing occasionally, and I want the reader to understand that it's probably a false alarm.